src/Model/NoDriverManager.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Model;
+
+use Sonata\CoreBundle\Model\ManagerInterface;
+use Sonata\MediaBundle\SonataMediaBundle;
+
+/**
+ * @internal
+ */
+class NoDriverManager implements ManagerInterface
+{
+    public function getClass()
+    {
+        SonataMediaBundle::noDriverRuntimeException();
+    }
+
+    public function findAll()
+    {
+        SonataMediaBundle::noDriverRuntimeException();
```

Comment:
Why don't we generate a new Exception class for this?

Reply:
This error will be shown to developer in a browser, if a pack with additional configuration was not installed. Here is a screenshot how it is decorated by `symfony/debug` in dev environment:

![Error message](http://joxi.ru/LmG4kG6teRgNZ2.jpg)

That message could be updated! We could add instructions how to install a pack =)
But, i agree, that method should be removed. Can i use `Sonata\CoreBundle\Exception\InvalidParameterException`?

Reply:
I thought more about new exception class, `NoDriverException`?

Reply:
Added new `NoDriverException`